src/Xablu.WebApiClient/WebApiClient.cs

```diff
@@ -22,12 +22,10 @@ public class WebApiClient : IWebApiClient
         private Lazy<HttpClient> _background;
         private Lazy<HttpClient> _userInitiated;
         private Lazy<HttpClient> _speculative;
-        private Func<HttpMessageHandler> httpHandler;
+        private Func<HttpMessageHandler> httpHandler = () => new HttpClientHandler();
```

Comment:
Right, good suggestion. I just didn't touch that for now because it already was like that.